### ISSUE TITLE
Missing file "nested_sampling_module.mod" when compiling

### ISSUE THREAD

**Issue by qminor** [JohannesBuchner/PolyChord#4]
When I ran "make" to compile, I get the following error:

InitSampler.f90:12.8:

    use nested_sampling_module,   only: NestedSampling
        1
Fatal Error: Can't open module file 'nested_sampling_module.mod' for reading at (1): No such file or directory

It seems there is a Fortran module file missing? Please let me know how this can be fixed.

Thanks,
Quinn

**Comment by JohannesBuchner** [comment]
Please don't use this repository, and use the newer upstream one instead.

**Comment by JohannesBuchner** [comment]
http://ccpforge.cse.rl.ac.uk/gf/project/polychord/

**Comment by ziotom78** [comment]
Is it true that the new repository at "cppforge" requires the user to register and having his registration request approved before being able to download the software?

**Comment by JohannesBuchner** [comment]
Yes. They also don't have a FOSS license as far as I understand it.
